src/components/fields/ArrayField.js

```diff
@@ -93,6 +100,78 @@ class ArrayField extends Component {
     const {items} = this.state;
     const {fields, definitions} = this.props.registry;
     const {SchemaField} = fields;
+    if (isFixedItems(schema)) {
+      const itemSchemas = schema.items.map(item => retrieveSchema(item, definitions));
+      const additionalSchema = allowAdditionalItems(schema) ? retrieveSchema(schema.additionalItems, definitions) : null;
+      return (
+          <fieldset
+              className={`field field-array field-array-fixed-items`}>
+            {title ? <legend>{title}</legend> : null}
+            {schema.description ?
+                <div className="field-description">{schema.description}</div> : null}
+            <div className="row array-item-list">{
+              itemSchemas.map((itemSchema, index) => {
+                const itemErrorSchema = errorSchema ? errorSchema[index] : undefined;
+                const itemIdPrefix = idSchema.id + "_" + index;
+                const itemIdSchema = toIdSchema(itemSchema, itemIdPrefix, definitions);
+                return (
+                    <div key={index}>
+                      <div className="col-xs-12">
+                        <SchemaField
+                            schema={itemSchema}
+                            uiSchema={uiSchema.items}
+                            formData={items[index]}
+                            errorSchema={itemErrorSchema}
+                            idSchema={itemIdSchema}
+                            required={this.isItemRequired(itemSchema)}
+                            onChange={this.onChangeForIndex(index)}
+                            registry={this.props.registry}/>
+                      </div>
+                    </div>
+                );
+              })
+            }{
+              additionalSchema && items && items.length <= itemSchemas.length ? null :
+                items.slice(itemSchemas.length).map((item, index) => {
```

Comment:
Nit: for legibility, it would be nice to store the sliced items list into some `fixedItems` intermediary const.